network/src/noise_wrapper/fuzzing.rs

```diff
@@ -8,7 +8,7 @@
 // This fuzzes the wrappers we have around our Noise library.
 //
 
-use crate::noise_wrapper::NoiseWrapper;
+use crate::noise_wrapper::NoiseUpgrader;
```

Comment:
if you change the name of the struct, should you change the name of the module as well?